<ui version="4.0" >
 <author>Gordos Kund</author>
 <class>TDP</class>
 <widget class="QDialog" name="TDP" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Print Preview</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QToolButton" name="zoomInToolButton" >
        <property name="toolTip" >
         <string>Zoom in</string>
        </property>
        <property name="text" >
         <string>+</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomOutToolButton" >
        <property name="toolTip" >
         <string>Zoom Out</string>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/minus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>145</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="page_SB" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Type page number here</string>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="keyboardTracking" >
         <bool>false</bool>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>271</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGraphicsView" name="graphicsView" >
     <property name="renderHints" >
      <set>QPainter::TextAntialiasing</set>
     </property>
     <property name="dragMode" >
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QToolButton" name="setupToolButton" >
        <property name="toolTip" >
         <string>Page Settings</string>
        </property>
        <property name="text" >
         <string>Setup</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/setup.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>31</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="fontSize_SB" >
        <property name="toolTip" >
         <string>Font Size</string>
        </property>
        <property name="minimum" >
         <number>-10</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="cellSize_SB" >
        <property name="toolTip" >
         <string>Cell Size</string>
        </property>
        <property name="minimum" >
         <number>-50</number>
        </property>
        <property name="maximum" >
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="grid_CB" >
        <property name="text" >
         <string>Grid</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apply_PB" >
        <property name="text" >
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>91</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okPdfButton" >
        <property name="toolTip" >
         <string>Save PDF</string>
        </property>
        <property name="text" >
         <string>Pdf</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/pdf.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton" >
        <property name="toolTip" >
         <string>Print Document</string>
        </property>
        <property name="text" >
         <string>Print</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/printer.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton" >
        <property name="toolTip" >
         <string>Cancel</string>
        </property>
        <property name="text" >
         <string>Cancel</string>
        </property>
        <property name="icon" >
         <iconset resource="jcn.qrc" >:/images/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>zoomInToolButton</tabstop>
  <tabstop>zoomOutToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../tableprinterresource.qrc" />
  <include location="jcn.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>TDP</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>565</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel" >
     <x>470</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>TDP</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>480</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
